src/main/java/be/kakumi/kachat/utils/Placeholder.java

```diff
@@ -1,13 +1,15 @@
 package be.kakumi.kachat.utils;
 
+import be.kakumi.kachat.models.Channel;
 import org.bukkit.entity.Player;
 
 public interface Placeholder {
     /***
      * Replace placeholder inside the message and chat format.
      * @param player Player who sent the message
+     * @param channel Channel in which the message should be sent
      * @param message Message to send (format + message)
      * @return Message to send with placeholder replace
      */
-    String format(Player player, String message);
+    String format(Player player, Channel channel, String message);
```

Comment:
This is for the KAChatAPI to use the correct channel to get the information for the placeholders. I figured out it was better to drill down the param instead of making the logic again inside the format, but.. then for the OverrideSymbolFormatter I just did the logic there. Which approach do you think it's better here? I don't know if the Placeholder interface has this shape for any particular reason and if we should try to avoid changing it.